client/components/Map/RequestDetail.jsx

```diff
@@ -102,6 +102,10 @@ class RequestDetail extends React.Component {
       address,
     } = pinsInfo[requestId];
 
+    // regex fix to replace "/" in typeName
```

Comment:
This is excellent, however we may want to use a more descriptive name for the new typeName. Instead of `replacedChar` perhaps something like `formattedTypeName` like so. And I think we could just use ' ' in place of replaceString in the second parameter to split (no need for variable). Thanks Shannan!

// regex fix to replace "/" in typeName
const formattedTypeName = typeName.split('/').join(' ');

Reply:
This sounds better, Thanks!